app/views/organizations/pets/show.html.erb

```diff
@@ -67,6 +67,22 @@
                   </div>
                 </div>
               </li>
+              <li class="list-group-item px-0">
+                <div class="d-flex justify-content-between
+                               align-items-center">
+                  <div class="d-flex align-items-center">
+                    <i class="fe fe-book me-2"></i>
+                    <div class="ms-2">
+                      <h5 class="mb-0 text-body">Placement Type</h5>
+                    </div>
+                  </div>
+                  <div>
+                    <div>
+                      <p class="text-dark mb-0 fw-semibold"><%= @pet.placement_type %></p>
```

Comment:
Can we capitalize the first letter? May as well do the `Sex` field while you are at it :)